build.gradle

```diff
@@ -13,7 +13,7 @@ buildscript {
 
 allprojects {
     group = 'com.github.niqdev'
-    version = '1.3.2'
+    version = '1.3.3'
```

Comment:
please revert this, I will change it when release the lib

Reply:
Okay

Reply:
Actually, It could be done by gradle task.

```groovy
def getVersionCode = { ->
  try {
    def code = new ByteArrayOutputStream()
    exec {
      commandLine 'git', 'rev-list', 'HEAD', '--count'
      standardOutput = code
    }
    return Integer.parseInt(code.toString().trim())
  } catch (exception) {
    return "1";
  }
}

// Version name is Last Tag Name + No. of commits form last Tag +  short git sha
def getVersionName = { ->
  try {
    def stdout = new ByteArrayOutputStream()
    exec {
      commandLine 'git', 'describe', '--tags', '--dirty'
      standardOutput = stdout
    }
    return stdout.toString().trim()
  } catch (exception) {
    return "0.0.0.1";
  }
}

// Use
android{
  defaultConfig {
    ...
    versionCode getVersionCode()
    versionName getVersionName()
    ...
  }
}
```

Reply:
nice, so to use it properly i need to tag it just before release the lib i.e. `./gradlew build bintrayUpload`. correct? can you add it in a different PR unless you think is going to be different in #59?

Reply:
hi @niqdev
I will do another PR, for that change.

Reply:
Right, this should be set automatically.

Reply:
Here you go #115